omnibus/files/private-chef-cookbooks/private-chef/templates/default/pedant_config.rb.erb

```diff
@@ -61,7 +61,11 @@ default_orgname <%= @default_orgname ? "\"#{@default_orgname}\"" : "nil" %>
 # testing location, you should not specify a value for this parameter.
 # The tests will still run, albeit slower, as they will now need to
 # poll for a period to ensure they are querying committed results.
+<% if node['private_chef']['search_provider'] == "solr" %>
 search_server "<%= @solr_url %>"
+<% else %>
+# search_server "<%= @solr_url %>"
```

Comment:
I didn't see anything in the test changs to handle no server url - will its absence cause trouble for pedant?